ultravox/training/config_base.py

```diff
@@ -20,6 +38,15 @@ class TrainConfig:
     # audio encoder model to use
     audio_model: str
 
+    # The data_dicts field complements data_sets, allowing for the inclusion of
+    # new datasets in the config.
+    #
+    # Due to simple_parsing's lack of support for containers of dataclass types,
```

Comment:
Is this true? I thought we had managed to solve this previously: https://github.com/lebrice/SimpleParsing/blob/master/examples/container_types/README.md

Reply:
Yes, the issue is specific to containers of dataclass types. The error message says something like "support for containers of dataclass types is not supported yet"